src/java.base/windows/native/libnio/ch/wepoll.c

```diff
@@ -858,11 +858,6 @@ int err_check_handle(HANDLE handle) {
 
 #define unused_var(v) ((void) (v))
 
-/* Polyfill `inline` for older versions of msvc (up to Visual Studio 2013) */
-#if defined(_MSC_VER) && _MSC_VER < 1900
-#define inline __inline
-#endif
-
```

Comment:
This is 3rd party code best to leave it out of this change.

Reply:
I agree, let's leave 3rd party code alone. I adjusted wepoll.c and kept the _MSC_VER check.